core/themes/basis/css/skin.css

```diff
@@ -1226,3 +1226,48 @@ div.error:before {
   /* background-image: -webkit-linear-gradient(transparent, transparent), url(../../../misc/message-error.svg); */
   /* background-image: linear-gradient(transparent, transparent), url(../../../misc/message-error.svg); */
 }
+
+/* Card view on default home page */
+
+.block-views-card-grid-block {
+  margin-top: -2em;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  background-color: #f2f2f2;
```

Comment:
This color isn't used in Basis so far. Even if this section would get processed by the Color module, there won't be any replacement for `#f2f2f2`.